doc/Overview/Mvux/Advanced/InspectGeneratedCode.md

```diff
@@ -0,0 +1,26 @@
+---
+uid: Overview.Mvux.Advanced.InspectGeneratedCode
+---
+
+# Inspecting the generated code
+
+MVUX makes extensive use of code generation. It's used to generate Bindable Proxy Models and entities and commands.  
+Inspecting the generated code can give you a lot of insight into how the Bindable Proxies work under the hood and can also improve your knowledge about MVUX operates behind the scenes and familiarize yourself with its internals.
+
+Viewing the generated code can be achieved in several ways:
+
+1. Placing the cursor on the class name and hitting <kbd>F12</kbd>:
```

Comment:
Do the following shortcuts assume using standard keybindings and running inside Visual Studio?  
Does this need to be made clear?  
What about people using VSCode? (or another editor?)